src/NuGet.Clients/PackageManagement.VisualStudio/Utility/ServiceLocator.cs

```diff
@@ -47,40 +47,54 @@ public static void InitializePackageServiceProvider(IServiceProvider provider)
             }
         }
 
+        public static async Task<TService> GetInstanceSafeAsync<TService>() where TService : class
```

Comment:
>GetInstanceSafeAsync [](start = 43, length = 20)

Let's do some analysis here, if we are moving to async loading of our package, can we eliminate this method completely?